rmgpy/solver/base.pyx

```diff
@@ -920,6 +920,15 @@ cdef class ReactionSystem(DASx):
                     # Set the maximum bi/trimolecular rate by using the user-defined rate constant threshold
```

Comment:
You could have a gas phase simulation that was constantVolume.  You should instead check what the object is, probably with isinstance(self,LiquidReactor).

Reply:
`LiquidReactor` and `SimpleReactor` now have methods to generate the threshold rate constants.